src/dev/impl/DevToys/Helpers/CertificateHelper.cs

```diff
@@ -68,6 +70,38 @@ internal static bool TryDecodeCertificate(string input, string? password, out st
                 return false;
             }
 
+            // Try to decode the X.509 extensions. Put this is a separate try/catch because it's not critical.
+            try
+            {
+                decoded += Environment.NewLine;
+
+                foreach (X509Extension x509Extension in certificate.Extensions)
+                {
+                    StringBuilder extensionData = new StringBuilder();
+                    AsnEncodedData asnEncodedData = new AsnEncodedData(x509Extension.Oid, x509Extension.RawData);
+
+                    // Add the name in brackets to match the previous output from X509Certificate.ToString()
+                    extensionData.Append($"[{x509Extension.Oid.FriendlyName}]");
+                    extensionData.Append(Environment.NewLine);
```

Comment:
Could you have used [StringBuilder.AppendLine](https://learn.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.appendline?view=net-7.0) here instead?

Reply:
Yes!

Reply:
457bd325d0b47555d917b6530109b254ca108bea